.github/workflows/CI.yml

```
@@ -18,6 +18,12 @@ jobs:
     - name: checkout
       uses: actions/checkout@v4
 
+    - name: Log in to Docker Hub
+      uses: docker/login-action@v2
```

actionlint (1.7.4)

the runner of "docker/login-action@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)